### DISCUSSION TITLE
Parte 2: sobre test hard-2

### DISCUSSION THREAD

**Discussion by domiSoto** [IIC2133-PUC/2024-2#143] [Tarea 2] [answered]
Holaa, en la parte 2 estoy pasando todos los test menos el hard-2, puede ser que el input.txt contenga algún code tree que no este completo por error?

**Comment by sebacarrascop** [comment]
hola!, lo vamos a revisar

**Comment by 4n1t4** [comment, marked as answer]
Hola!
El test no tiene errores.

**Comment by 4n1t4** [comment]
Te recomiendo que verifiques que manejas bien las colisiones en todas las tablas que utilices. 
Exito en la tarea!
